deepchem/models/tensorgraph/layers.py

```diff
@@ -4303,3 +4303,23 @@ def batch_mat_mult(self, A, B):
     result = tf.matmul(A_reshape, B)
     result = tf.reshape(result, tf.stack([A_shape[0], A_shape[1], axis_2]))
     return result
+
+
+class Hingeloss(Layer):
+
+  def __init__(self, in_layers=None, **kwargs):
+    super(Hingeloss, self).__init__(in_layers, **kwargs)
+    try:
+      self._shape = self.in_layers[1].shape
+    except:
+      pass
+
+  def create_tensor(self, in_layers=None, set_tensors=True, **kwargs):
+    inputs = self._get_input_tensors(in_layers)
```

Comment:
Please document that in_layers should be lables, logits.

Also is labels one hot encoded or not here?

Reply:
yes the labels are one hot encoded here